app/services/metrics_api/v3/presenters/view_new.rb

```diff
@@ -11,15 +11,12 @@ def initialize(metric)
 
         def to_json
           {
-            id: @m.display_collection,
+            id: @m.record_field_value,
```

Comment:
Hey bro,
I assumed that you are not using  `view_new` from your `end_points`.
May be we can delete 😬 this file 😬

Reply:
Hi Yar, we have a follow-up story to delete all code related to the old metric logging.
thanks for pointing it out